docs/developer/validation-rules/camel-casing-validation-rules.md

```diff
@@ -1,6 +1,6 @@
 #M3016 - DefinitionsPropertiesNamesCamelCase/BodyPropertiesNamesCamelCase
 ## Description
-This violation is flagged if a model definition's property name (DefinitionsPropertiesNamesCamelCase) or a request body parameter's property name (BodyPropertiesNamesCamelCase)  is not in `camelCase` format. This is because the model's properties are sent across the wire and must adhere to common Json conventions for naming. This implies that every property name must start with a lower cased letter and all subsequent words within the name must start with a capital letter. In cases where there are abbreviations involved, a maximum of three contiguous capital letters are allowed. Eg: `redisCache`, `publicIPAddress`, `location` are valid, but `sampleSQLQuery` is not (must be renamed as `sampleSqlQuery`).
+This violation is flagged if a model definition's property name (DefinitionsPropertiesNamesCamelCase) or a request body parameter's property name (BodyPropertiesNamesCamelCase)  is not in `camelCase` format. This is because the model's properties are sent across the wire and must adhere to common Json conventions for naming. This implies that every property name must start with a lower cased letter and all subsequent words within the name must start with a capital letter. In cases where there are acronyms involved, a maximum of three contiguous capital letters are allowed (acronyms should not be longer that 2 characters - see https://msdn.microsoft.com/en-us/library/141e06ef(v=vs.71).aspx, third upper case could be start of next word). Eg: `redisCache`, `publicIPAddress`, `location` are valid, but `sampleSQLQuery` is not (must be renamed as `sampleSqlQuery`).
```

Comment:
Just want to make sure we are not adhering to the MSFT Json naming conventions here. Can you confirm this.